src/main/java/duke/Parser.java

```diff
@@ -0,0 +1,76 @@
+package duke;
+
+import duke.exception.EmptyDescriptionException;
+import duke.exception.MissingParameterException;
+import duke.exception.WrongCommandException;
+
+public class Parser {
+
+    public static String parseInput(String command) {
+        return command.toLowerCase().trim();
+    }
+
+    public static int parseNumber(String command) throws EmptyDescriptionException, NumberFormatException {
+        if (command.split(" ").length < 2) {
+            throw new EmptyDescriptionException();
+        }
+        return Integer.parseInt(command.split(" ")[1]);
+    }
+
+
+    public static String[] parseTask(String command, Action taskType) throws EmptyDescriptionException, MissingParameterException {
```

Comment:
Where it is not obvious what the code is doing, provide method descriptions in [javadocs style](https://se-education.org/guides/conventions/java/basic.html#comments) to explain what the method is doing.